bower.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "angular-evaporate",
+  "homepage": "https://github.com/uqee/angular-evaporate",
+  "description": "AngularJS module for the EvaporateJS library",
+  "authors": [
+    "Denis Zhbankov <denis.zhbankov@gmail.com>"
+  ],
+  "keywords": [
+    "upload",
+    "resumable upload",
+    "evaporate",
+    "angular"
+  ],
+  "version": "2.1.2",
```

Comment:
Property `version` is [deprecated and ignored](https://github.com/bower/spec/blob/master/json.md#version) by `bower`, so it shouldn't be used.